### PR TITLE
Digest summary

### PR DESCRIPTION
**Pull request by mattiaverga**
What about to add a short summary of the messages at the top of the digest when messages are set to verbose mode?
This can help to have a quick look of digest's contents without losing the details in the email.

### CODE REVIEW

File: fmn/consumer/backends/mail.py [hunk 1]
```diff
@@ -148,8 +148,17 @@ def _format_line(msg):
 
         n = len(queued_messages)
         subject = u"Fedora Notifications Digest (%i updates)" % n
+        summary = u"Digest summary:\n"
```
Comment (ralphbean) [inline fmn/consumer/backends/mail.py]: A request:  Can we only prepend the summary if ``verbose`` is False?  It seems like the only substantive difference between the summary and the actual body is the possible presence of ``long_form`` content.

Comment (mattiaverga) [inline fmn/consumer/backends/mail.py]: Sorry, I do not understand the request.
At the moment the patch creates a "summary" text and prepend it to the usual digest content if the digest is set to Verbose full, like:
`Digest summary:
1.  First message header
2. Second message header
-----------------------------------------------
<usual digest content with Verbose = Full>
`

If the digest is set to Verbose = false it does nothing (it creates the "summary" text but doesn't prepend to email content.

Do you mean you want to prepend only the "Digest Summary" text to the usual content if Verbose = false? Like
`Digest summary:
<usual digest content with Verbose = False>
`

Comment (ralphbean) [inline fmn/consumer/backends/mail.py]: My apologies - I mis-read the original diff.  It is good to merge as written.

**Comment by ralphbean** [comment]
:+1: from me.  Thanks again @mattiaverga!

**Pull request merged**